packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1299,7 +1389,11 @@ class Auth {
      * A wrapper method for the SLAS endpoint: resetPassword.
      *
      */
-    async resetPassword(parameters: ShopperLoginTypes.PasswordActionVerifyRequest) {
+    async resetPassword(parameters: {
```

Comment:
Similar comment from above to here. I wonder if we want to define a new type for these parameters?

Reply:
Same approach as above, I'm guessing the intention is to simplify the API to only require the essential user inputs (pwd_action_token, new_password, user_id). The function automatically provides client_id and channel_id from the configured settings.

About defining a new type: My opinion is since these are simple parameter objects used in just here, inline types keep the API straightforward. If we had multiple methods using the same type, then a named type would make sense.